introduction.md

```diff
@@ -1,6 +1,12 @@
 Dojo, I am Elmer.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?